sixel.go

```diff
@@ -1,69 +1,69 @@
 package main
 
 import (
-	"fmt"
+	"log"
 	"os"
-	"strings"
+	"strconv"
 
 	"github.com/gdamore/tcell/v2"
 )
 
 const (
 	gSixelBegin = "\033P"
-
-	// The filler character should be:
-	// - rarely used: the filler is used to trick tcell into redrawing, if the
-	//   filler character appears in the user's preview, that cell might not
-	//   be cleaned up properly
-	// - ideally renders as empty space: the filler alternates between bold
-	//   and regular, using a non-space would look weird to the user.
-	gSixelFiller = '\u2000'
 )
 
 type sixelScreen struct {
-	xprev, yprev int
-	sixel        *string
-	altFill      bool
-	lastFile     string // TODO maybe use hash of sixels instead to flip altFill
+	lastFile string
+	lastSxW  int
+	lastSxH  int
+	lastWinW int
+	lastWinH int
 }
 
-func (sxs *sixelScreen) fillerStyle(filePath string) tcell.Style {
-	if sxs.lastFile != filePath {
-		sxs.altFill = !sxs.altFill
-	}
-
-	if sxs.altFill {
-		return tcell.StyleDefault.Bold(true)
-	}
-	return tcell.StyleDefault
-}
+func (sxs *sixelScreen) unlockSixel(win *win, screen tcell.Screen, filePath string) {
+	if filePath != "" && (filePath != sxs.lastFile || win.w != sxs.lastWinW || win.h != sxs.lastWinH) {
```

Comment:
The check should be `sxs.lastFile != ""` as it represents whether there was a previous sixel image drawn or not. `filePath` is the path of the newly selected file, and is practically never blank.

```suggestion
	if sxs.lastFile != "" && (filePath != sxs.lastFile || win.w != sxs.lastWinW || win.h != sxs.lastWinH) {
```